netdata/kernel-collector
# Sync branch with `netdata/libbpf`

##### Summary
This PR is syncing current `repo` with `netdata/libbpf`.
 
##### Test Plan
1. Clone branch
2. Run `make` in root directory
3. Enter `co-re` directory, and run `make`

We expect to compile `legacy` code and `co-re` code without any problems.
##### Additional information

Another possibility is to use docker image instead to run step `2`.
